example/spec/parallel/syncGranule/SyncGranuleDuplicateHandlingSpec.js

```diff
@@ -278,18 +277,18 @@ describe('When the Sync Granule workflow is configured', () => {
       it('captures both files', async () => {
         // This assertion is to check that the granule has been updated in dynamo
```

Comment:
Nit: this comment should probably be more generic (e.g. dynamo -> datastore), given we're now querying the API - with the change to reading from the API being done here we won't catch it in the granules ticket.